documentation/src/main/asciidoc/mapstruct-reference-guide.asciidoc

```diff
@@ -133,9 +133,36 @@ apply plugin: 'net.ltgt.apt-eclipse'
 
 dependencies {
     ...
-    compile "org.mapstruct:mapstruct:${mapstructVersion}"
+    implementation "org.mapstruct:mapstruct:${mapstructVersion}"
+    annotationProcessor "org.mapstruct:mapstruct-processor:${mapstructVersion}"
 
+    // If you are using mapstruct in test code
+    testAnnotationProcessor "org.mapstruct:mapstruct-processor:${mapstructVersion}"
+}
+...
+----
+====
+Gradle (3.3 and older)
```

Comment:
I think that a dot (.) is missing here in order for the text to look like a caption. Perhaps name it `.Gradle configuration (3.3 and older)`?

Reply:
Yes, you are right, sorry for that :$
And about the mapstruct-on-gradle, of course, I will help you with that as well